workbench-core/accounts/src/services/accountService.ts

```diff
@@ -175,4 +177,58 @@ export default class AccountService {
 
     return accountMetadata.id;
   }
+
+  private async _getAccountWithoutMetadata(accountId: string): Promise<Account> {
+    const accountEntry = (await this._aws.helpers.ddb
+      .get({ pk: `${resourceTypeToKey.account}#${accountId}` })
+      .execute()) as GetItemCommandOutput;
+
+    if (!accountEntry.Item) {
+      throw Boom.notFound(`Could not find account ${accountId}`);
+    }
+
+    return accountEntry.Item! as unknown as Account;
+  }
+
+  private async _getAccountWithMetadata(accountId: string): Promise<Account> {
+    const pk = `ACC#${accountId}`;
+
+    const data = await this._aws.helpers.ddb.query({ key: { name: 'pk', value: pk } }).execute();
+
+    if (data.Count === 0) {
+      throw Boom.notFound(`Could not find account ${accountId}`);
+    }
+
+    const items = data.Items!.map((item) => {
+      return item;
+    });
+    let account: Account = {
+      name: '',
+      awsAccountId: '',
+      encryptionKeyArn: '',
+      envMgmtRoleArn: '',
+      environmentInstanceFiles: '',
+      hostingAccountHandlerRoleArn: '',
+      stackName: '',
+      status: '',
+      subnetId: '',
+      vpcId: ''
+    };
+    for (const item of items) {
+      // parent environment item
+      const sk = item.sk as unknown as string;
+
+      if (sk === pk) {
+        account = { ...account, ...item };
+        continue;
+      }
+
+      const associationPrefix = sk.split('#')[0];
+
```

Comment:
can we add this prefix to `resourcetypeKey.CostCenter` if it doesnt exist and use it instead

Reply:
fixed